libs/knowledge-store/ragstack_knowledge_store/links.py

```diff
@@ -35,20 +35,20 @@ def __init__(self, kind: str, tag: str) -> None:
         super().__init__(kind=kind, tag=tag, direction="bidir")
 
 
-LINK_TAGS = "link_tags"
+LINK_SET = "link_set"
```

Comment:
Maybe just `LINKS` ?

Reply:
Could be links. Wasn't sure if we wanted to try to keep it distinct from other things. I guess we could do something like `GRAPH_LINKS`... butI'd be ok with `LINKS`.